scalastyle-config.xml

```diff
@@ -67,29 +66,35 @@ This file is divided into 3 sections:
         </parameters>
     </check>
 
-    <check level="error" class="org.scalastyle.scalariform.SpacesAfterPlusChecker" enabled="true"></check>
+    <check level="error" class="org.scalastyle.scalariform.SpacesAfterPlusChecker" enabled="true"/>
 
-    <check level="error" class="org.scalastyle.scalariform.SpacesBeforePlusChecker" enabled="true"></check>
+    <check level="error" class="org.scalastyle.scalariform.SpacesBeforePlusChecker" enabled="true"/>
 
-    <check level="error" class="org.scalastyle.file.WhitespaceEndOfLineChecker" enabled="true"></check>
+    <check level="error" class="org.scalastyle.file.WhitespaceEndOfLineChecker" enabled="true"/>
 
     <check level="error" class="org.scalastyle.file.FileLineLengthChecker" enabled="true">
         <parameters>
             <parameter name="maxLineLength"><![CDATA[120]]></parameter>
-            <parameter name="tabSize"><![CDATA[2]]></parameter>
-            <parameter name="ignoreImports">true</parameter>
+            <parameter name="tabSize"><![CDATA[4]]></parameter>
+            <parameter name="ignoreImports"><![CDATA[false]]></parameter>
         </parameters>
     </check>
 
     <check level="error" class="org.scalastyle.scalariform.ClassNamesChecker" enabled="true">
         <parameters>
-            <parameter name="regex"><![CDATA[[A-Z][A-Za-z]*]]></parameter>
+            <parameter name="regex"><![CDATA[^[A-Z][A-Za-z]*$]]></parameter>
```

Comment:
Scala allows underscores, number characters and non-latin characters. I don't have a concrete example, but I can see valid use for adding `0-9` to the second group in the pattern.

Reply:
`^[A-Z][A-Za-z]*$` is just scalastyle's default value.
in case needed in future, we can make the pattern more lenient.